src/controllers/v1/translator.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import dotenv from 'dotenv';
+import Response from '../../utils/response';
+
+dotenv.config();
+
+const translatorController = async (req, res) => {
+  let text = req.body.text || '';
+  const lang = req.body.lang;
+
+  if (text.length > 280) {
+    return Response.inputError(res, 'text should be no more than 280 characters');
+  }
```

Comment:
Lets strive to keep our controllers as pure as possible. All this validation logic should be handled in some middleware. Also `text` should never be whitespace and it should be a const